start_cerberus.py

```diff
@@ -125,11 +124,14 @@ def main(cfg):
             if not server_status:
                 api_fail_count += 1
 
-            for node in master_nodes:
-                taint = kubecli.get_taint_from_describe(node)
-                if "none" in str(taint).lower() or "NoSchedule" not in str(taint):
-                    logging.info("Iteration %s: Master node %s has incorrect scheduling taint, "
-                                 "%s " % (iteration, node, str(taint)))
+            # Check for NoSchedule taint in all the master nodes once in every 10 iterations
+            if iteration == 1 or not iteration % 10:
```

Comment:
Will this check the master nodes taint the 0 and 1st iteration and then the 10th one every other time? Is that what you intended? I think `if not iteration % 10:`will work

Reply:
The iteration starts from 1, so we will have to wait till 10th iteration for the first master_taint check. However, I have changed it to `if iteration % 10 == 1:` so that master_taint check is executed on iterations 1, 11, 21 and so on.